assets/js/modules/idea-hub/components/settings/SettingsView.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Idea Hub Settings View component
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { sanitizeHTML } from '../../../../util';
+import { STORE_NAME } from '../../datastore/constants';
+const { useSelect } = Data;
+
+export default function SettingsView() {
+	const dashboardPermalink = useSelect( ( select ) => select( STORE_NAME ).getAdminScreenURL() );
+
+	const content = sprintf(
+		/* translators: %s is the URL to the Site Kit dashboard. */
+		__( 'To view ideas for new content, <a href="%s">visit the dashboard</a>', 'google-site-kit' ),
+		dashboardPermalink
+	);
+
+	return (
+		<p
+			dangerouslySetInnerHTML={ sanitizeHTML( content, {
+				ALLOWED_TAGS: [ 'a' ],
+				ALLOWED_ATTR: [ 'href' ],
+			} ) }
+		/>
```

Comment:
We need to use `createInterpolateElement`: https://github.com/google/site-kit-wp/blob/80868c0780a1d9febc153adadea50617624a22db/assets/js/components/OptIn.js#L77-L88

Reply:
Updated in https://github.com/google/site-kit-wp/pull/3557/commits/e0b9a129ce24f2b3146d3d5f4e83fcd7385f0a4c